buzzmobile/simulation/models/car/car_joints.urdf

```diff
@@ -356,7 +356,7 @@
     <joint name="lidar_joint" type="fixed">
         <parent link="chassis"/>
         <child link="laser"/>
-        <origin xyz="0.255 0 5" rpy="0 0 0"/>
+        <origin xyz="2 0 0" rpy="0 0 0"/>
```

Comment:
is this the correct location of the lidar?

if not can you make a TODO and an issue to mesure it in the car exactly?